woodwork/statistics_utils/_cast_nullable_int_and_datetime_to_int.py

```diff
@@ -0,0 +1,12 @@
+from woodwork.logical_types import IntegerNullable
+
+
+def _cast_nullable_int_and_datetime_to_int(data_dict, woodwork_columns):
+    for col_name in data_dict:
+        column = woodwork_columns[col_name]
+        if isinstance(column.logical_type, IntegerNullable):
+            cur_dtype = data_dict[col_name].dtype
+            new_dtype = cur_dtype.name.lower()  # e.g. Int64 -> int64
+            data_dict[col_name] = data_dict[col_name].astype(new_dtype)
```

Comment:
since `IntegerNullable`'s physical type is always "Int64" that would be sufficient, but I think this if statement should be re-written for a more generic check for nullable int physical types instead of `IntegerNullable` specifically